src/components/body/content/lessons/Lessons.tsx

```diff
@@ -1,28 +1,55 @@
 import "./index.scss";
+import { 
+  FC, useEffect, useState 
+} from "react";
 import { FormattedMessage } from "react-intl";
 import LessonCard from "../../../LessonCard";
-import React from "react";
-import items from "../../../../assets/lessonCard.json";
 
-const Lessons: React.FC = () => {
+const Lessons: FC = () => {
+  const [lessons, setLessons] = useState([]);
```

Comment:
Import lesson type from lesson component and use it for lessons, e.g.

const [lessons, setLessons] = useState&lt;LessonType&gt;([]);

Reply:
done